src/config/model.h

```diff
@@ -24,7 +24,7 @@ extern const char MODEL_TEMPLATE[];
 #define UNKNOWN_ICON ("media/noicon" IMG_EXT)
 
 //This cannot be computed, and must be manually updated
-#define NUM_PROTO_OPTS 6
+#define NUM_PROTO_OPTS 8
```

Comment:
I think MAX_PROTO_OPTS is a better name

Reply:
Ok, but I'll do that in another PR as this requires to modify lots of protocol files.